### PR TITLE
unify types and fix some warnings (#223)

### LINKED ISSUE #223: Uwspólnić typy między frontem i bakiem
Uwspólnić typy między serwerem a frontami, by nie musiec ich rzucać jeden na drugi

### PR DESCRIPTION
**Pull request by ostatni5**

### CODE REVIEW

**Review by michal367 — APPROVED** [review]
Great

File: src/lecturer/components/reactionReceiveView/ReactionCounter.tsx [hunk 1]
```diff
@@ -30,7 +30,8 @@ export function ReactionCounter(props: ReactionCounterProps) {
         setTimeout(() => {
             setChanged(true);
         },150)
-    }, [store.reactionValues[props.index]]);
```
Comment (Derstilon) [inline src/lecturer/components/reactionReceiveView/ReactionCounter.tsx:33]: The animation now plays for every reaction. Before it was only for the one that changed :/
image

**Review by Derstilon — CHANGES_REQUESTED** [review]

File: src/lecturer/components/reactionReceiveView/ReactionCounter.tsx [hunk 2]
```diff
@@ -30,7 +30,8 @@ export function ReactionCounter(props: ReactionCounterProps) {
...
+    }, [props.index, store.reactionValues]);
```
Comment (ostatni5) [inline src/lecturer/components/reactionReceiveView/ReactionCounter.tsx:33]:
```suggestion
    }, [store.reactionValues[props.index]]);
```

**Review by Derstilon — APPROVED** [review]
Works just fine

**Pull request merged**